core/services/wifi/main.py

```diff
@@ -143,6 +143,24 @@ async def disconnect() -> Any:
     logger.info("Succesfully disconnected from network.")
 
 
+@app.post("/hotspot", summary="Enable/disable hotspot.")
+@version(1, 0)
+def toggle_hotspot(enable: bool, ssid: Optional[str] = None, password: Optional[str] = None) -> Any:
+    try:
+        wifi_manager.enable_hotspot(enable, ssid, passphrase=password)
```

Comment:
There is no more `enable` in `enable_hotspot`, how did the linters not get it ?

Reply:
They got it. It's solved on the following commit. I will merge them when finished.

Reply:
My bad. Committed that fix in the wrong commit.